skewer.yaml

```diff
@@ -91,20 +92,19 @@ steps:
         - run: skupper link generate > link.yaml
```

Comment:
why not issue token?

Reply:
We should recommend using links on system sites, because if an AccessToken is used,
an eventual reload of the system site might fail, as the AccessToken can no longer be valid.

I will add a note explaining it.

Reply:
Done.